docs/paper/dev/misc/reading-stacktraces.md

```diff
@@ -0,0 +1,51 @@
+# Reading Stacktraces
+
+## What is a stacktrace?
+In Java, a stacktrace shows the call stack of a thread. The call stack is the path of execution that led to the current point in the program.
+Usually, the stacktrace will be printed to the console when an exception is not handled correctly.
+
+Stacktraces are a useful tool for debugging your code. They show you the exact line of code that caused an error, and the 
+line of code that called that line of code, and so on. This is useful because it allows you to see the exact path of execution that led to the error.
+
+### Example:
+
+Here is an example of a stacktrace which has been caused due to a `NullPointerException`:
+
+```
+[15:20:42 ERROR]: Could not pass event PluginEnableEvent to TestPlugin v1.0
```

Comment:
I would include a blurb talking about this line. While it isn't in every stacktrace, it feels common enough to say this means an exception happened while handling X event from Y plugin.